cinema_service_db_data.json

```diff
@@ -0,0 +1,550 @@
+      "duration": 151,
+      "genres": [
+        1,
+        2,
+        3
+      ],
+      "actors": [
+        1,
+        2,
+        3
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 2,
+    "fields": {
+      "title": "Inception",
+      "description": "A thief who steals corporate secrets through the use of dream-sharing technology is given the inverse task of planting an idea into the mind of a C.E.O.",
+      "duration": 148,
+      "genres": [
+        4,
+        5,
+        6
+      ],
+      "actors": [
+        2,
+        4,
+        5
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 3,
+    "fields": {
+      "title": "Looper",
+      "description": "In 2074, when the mob wants to get rid of someone, the target is sent into the past, where a hired gun awaits - someone like Joe - who one day learns the mob wants to 'close the loop' by sending back Joe's future self for assassination.",
+      "duration": 119,
+      "genres": [
+        2,
+        4,
+        6
+      ],
+      "actors": [
+        4,
+        6,
+        7
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 4,
+    "fields": {
+      "title": "Unbreakable",
+      "description": "A man learns something extraordinary about himself after a devastating accident.",
+      "duration": 106,
+      "genres": [
+        2,
+        6,
+        7
+      ],
+      "actors": [
+        6,
+        8,
+        9
+      ]
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 1,
+    "fields": {
+      "show_time": "2024-10-08T13:00:00Z",
+      "movie": 1,
+      "cinema_hall": 1
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 2,
+    "fields": {
+      "show_time": "2024-10-09T13:00:00Z",
+      "movie": 2,
+      "cinema_hall": 2
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 3,
+    "fields": {
+      "show_time": "2024-10-10T13:00:00Z",
+      "movie": 3,
+      "cinema_hall": 3
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 4,
+    "fields": {
+      "show_time": "2024-10-11T13:00:00Z",
+      "movie": 4,
+      "cinema_hall": 4
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 5,
+    "fields": {
+      "show_time": "2024-10-12T13:00:00Z",
+      "movie": 4,
+      "cinema_hall": 1
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 6,
+    "fields": {
+      "show_time": "2024-10-13T13:00:00Z",
+      "movie": 3,
+      "cinema_hall": 2
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 7,
+    "fields": {
+      "show_time":"2024-10-14T13:00:00Z",
+      "movie": 1,
+      "cinema_hall": 3
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 8,
+    "fields": {
+      "show_time": "2024-10-15T13:00:00Z",
+      "movie": 2,
+      "cinema_hall": 4
+    }
+  },
+  {
+    "model": "cinema.order",
+    "pk": 1,
+    "fields": {
+      "created_at": "2022-08-09T09:06:18.876Z",
```

Comment:
The `created_at` timestamps for orders are identical. Consider varying these timestamps to better simulate real-world data.